### PR TITLE
Beta

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Dummy PR

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
Just being on my phone, did you also update the search provider for user and groups and also the document provider?

**Comment by tobiasKaminsky** [comment]
Yes, but you can test it of course to make sure everything is working. 
I have tested it, but of course I could have overseen something

**Comment by AndyScherzinger** [comment]
Sure, will do. Most likely Monday or Tuesday. Don't have the time before that :(

File: AndroidManifest.xml [hunk 1]
```diff
@@ -63,6 +63,9 @@
     <uses-permission android:name="android.permission.READ_PHONE_STATE" />
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
 
+    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+
```
Comment (AndyScherzinger) [inline AndroidManifest.xml]: Isn't this now a double entry looking at lines 63/64?

Comment (tobiasKaminsky) [inline AndroidManifest.xml]: Not double, but triple ;)
Thanks for the catch!

File: build.gradle [hunk 2]
```diff
@@ -124,7 +137,12 @@ android {
         debug.setRoot('build-types/debug')
         release.setRoot('build-types/release')
     }
-	
+
+    dexOptions {
```
Comment (AndyScherzinger) [inline build.gradle]: Now the dex options are duplicated?! 0_o @tobiasKaminsky

File: src/com/owncloud/android/utils/FileStorageUtils.java [hunk 3]
```diff
@@ -133,13 +126,21 @@ public static String getTemporalPath(String accountName) {
...
+        } else {
+            StatFs stats = new StatFs(savePath.getAbsolutePath());
+            return stats.getAvailableBlocks() * stats.getBlockSize();
+        }
+
     }
     
     public static String getLogPath()  {
-        return MainApp.getStoragePath() + File.separator + MainApp.getDataFolder() + File.separator + "log";
+        return Environment.getExternalStorageDirectory() + File.separator + MainApp.getDataFolder() + File.separator + "log";
```
Comment (AndyScherzinger) [inline src/com/owncloud/android/utils/FileStorageUtils.java]: This is legacy and the removed code was the newer version (ExternalSD card)

File: build.gradle (hunk 2 above)
Comment (tobiasKaminsky) [inline build.gradle]: removed

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky can you do another beta release? This would then automatically include https://github.com/nextcloud/android-library/pull/21 :) So this could then be tested, works with the test account I got.

File: betaUpdateStatus [hunk 4]
```diff
@@ -1,15 +1,14 @@
...
-[07 / 13] Fix DavDroid integration append necessary addition to mURI (justin-sleep/upstream) [https://github.com/nextcloud/android/pull/401] no update needed
-[08 / 13] Updated federated sharing edit options (nextcloud/federated_sharing_edit_options) [https://github.com/nextcloud/android/pull/395] CONFLICT
-[09 / 13] Streaming (nextcloud/streaming) [https://github.com/nextcloud/android/pull/183] CONFLICT
-[10 / 13] Resize cache (nextcloud/resizeCache) [https://github.com/nextcloud/android/pull/144] CONFLICT
-[11 / 13] Instant upload in the context of multi account support (nextcloud/instant_upload_multi_account) [https://github.com/nextcloud/android/pull/98] CONFLICT
-[12 / 13] Resized images (nextcloud/resizedImages) [https://github.com/nextcloud/android/pull/69] CONFLICT
-[13 / 13] Master (nextcloud/master) [https://github.com/nextcloud/android] merged successfully
+2017-02-06
+Branches: 12
+[01 / 12] Rewrite auto upload (nextcloud/rewrite-auto-upload) [https://github.com/nextcloud/android/pull/615] CONFLICT
```
Comment (AndyScherzinger) [inline betaUpdateStatus]: @mario to get some feedback, can you try to manually merge (and resolving conflicts) this into the beta branch? That way we can probably get some feedback :)

**Comment by tobiasKaminsky** [comment]
Closing as this drives Codacy crazy ;-)

**Comment by tobiasKaminsky** [comment]
Do NOT remove beta branch!

**Pull request closed without merging**